src/components/Login.jsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { signInWithEmailAndPassword } from 'firebase/auth';
+import { auth } from '../firebaseConfig'; // Ensure this path is correct
+import '../styles/Login.css';
+
+const Login = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+
+  const navigate = useNavigate();
+
+  const handleLogin = async (e) => {
+    e.preventDefault();
+
+    try {
+      const userCredential = await signInWithEmailAndPassword(
+        auth,
+        email,
+        password,
+      );
+      const { user } = userCredential;
+
+      // Get the auth token (ID token)
+      const token = await user.getIdToken();
+
+      // Store the token in localStorage
+      localStorage.setItem('authToken', token);
+
+      // Redirect to the 'chatbot' page
+      setTimeout(() => navigate('/chatbot'), 500); // Updated path here
+    } catch (error) {
+      setError('Invalid email or password');
+    }
+  };
```

Comment:
@MARYAMM27  your handle login function has bug, when I click of the login button I get the following error:
And I am also getting the blank screen

![Screenshot 2024-10-14 134937](https://github.com/user-attachments/assets/e044c95e-7a27-481d-80ef-2aae8876836c)

Reply:
@emhamza "I have reviewed and corrected this. Please check it and let me know if everything is good."